Hazel/src/Hazel/Core/PlatformDetection.h

```diff
@@ -33,6 +33,9 @@
 	#error "Android is not supported!"
 #elif defined(__linux__)
 	#define HZ_PLATFORM_LINUX
+	// Linux is not offically supported by Hazel.
+	// You can remove this line to enable Linux support
+	#error "Linux is not supported! Read message in Hazel/src/Hazel/Core/PlatformDetection.h"
```

Comment:
I think this is misguided.

Reply:
I'm with DaveAxiom on this one...

Reply:
I'm confused on what you mean

Reply:
Your branch should work without strings attached. That's my take.